test/controllers/ht_registrations_controller_test.rb

```diff
@@ -414,6 +414,40 @@ def setup
     assert_equal "edit", @controller.action_name
   end
 
+  test "finalized user uses downcased shibboleth id" do
```

Comment:
@moseshll I added tests here since there weren't any existing tests for `RegistrationMover`, but I could certainly do that if you prefer.

Reply:
Yeah, 100% coverage in `RegistrationMover` in Coveralls but all from controller testing. I'd say create a new `RegistrationMover` test file -- we're sure to need it.